src/controls/extras/treeitem.coffee

```diff
@@ -34,16 +34,25 @@ class TreeItem #extends Base
   handle_relevant_or_required: ->
     tree_div = @tree.tree_root
     current_node = tree_div.find("[node_value = '" + @value + "']")
-
-    current_node.attr('relevant', @node_relevant())
+    current_node.attr('item-required', @node_required())
```

Comment:
I don't think we should update the core spec, but I think these sorts of extensions should be documented so we know how they're supposed to behave (and can detect brokenness), originators (SDPS) have a reference for how to format their data, and third parties can make use of them.

Isn't there an ECHO Forms schema? Does this interfere with that?